stats/opentelemetry/opentelemetry.go

```diff
@@ -171,6 +176,14 @@ func removeLeadingSlash(mn string) string {
 	return strings.TrimLeft(mn, "/")
 }
 
+func isMetricsDisabled(mo MetricsOptions) bool {
```

Comment:
The code now looks like:

```go
if !isMetricsDisabled() {
}
```

It should instead be:

```go
if isMetricsEnabled() {
}
```

to avoid a double negative.

Reply:
Updated